Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN poetry config virtualenvs.create false && poetry install
 
 FROM base as production
 COPY todo_app ./todo_app
-ENTRYPOINT ["poetry", "run", "gunicorn", "--bind", "0.0.0.0:80", "wsgi:run()"]
+CMD poetry run gunicorn --bind 0.0.0.0:${PORT-80} "wsgi:run()"
```

Comment:
It would be more common to use `${PORT:-80}`. The colon means it will also use 80 if the PORT string is empty